man/shadow.5.xml

```diff
@@ -199,22 +192,21 @@
 	</term>
 	<listitem>
 	  <para>
-	    The date of expiration of the account, expressed as the number
-	    of days since 1970-01-01 00:00:00 UTC.
+	    The date when the account expires,
+	    expressed as the number of days since 1970-01-01 00:00:00 UTC.
```

Comment:
The number of days should be since a given date, not a time.  Just like above (for consistency).